memsource/api.py

```diff
@@ -466,9 +466,18 @@ def _create(
         )
 
     def listByProject(self, project_id: int) -> List[models.JobPart]:
```

Comment:
```suggestion
    def listByProject(self, project_id: int, page: int = 0) -> List[models.JobPart]:
```

I think its better to accept the `page` parameter instead, I think getting all results should be responsible of the caller not the library 🙂